src/config/contracts.ts

```diff
@@ -314,3 +317,15 @@ export function getContract(chainId: number, name: string): string {
 
   return CONTRACTS[chainId][name];
 }
+
+function makeGetContract<T extends unknown & { abi: InterfaceAbi; connect: (address: string) => unknown }>(
```

Comment:
```suggestion
function makeGetContract<T extends { abi: InterfaceAbi; connect: (address: string) => unknown }>(
```

`unknown & {...}` is equals to `{...}` as unknown a supertype of any type, so there's no necessity to use unknown